### ISSUE TITLE
Auto Hot reload on save

### ISSUE THREAD

**Issue by shivahnshankar** [emacs-lsp/lsp-dart#56]
Hi
Having the hot reload and hot restart commands is wonderful! Is there an easy way to setup hot reload on every save of the active project files?

**Comment by ericdallo** [comment]
Thank you!
You mean during the debug, right? 
Currently no, as no one asked that feature before, but I think it can be implemented :)

**Comment by shivahnshankar** [comment]
Yes during debug..thanks

**Comment by ericdallo** [comment]
Ok, I'll try to offer a minor-mode that when saving buffer it hot-reloads the code

**Comment by ericdallo** [comment]
Hey @shivahnshankar, that was implemented on latest `lsp-dart`, `1.15.1`.
You can enable via:
```elisp
(setq lsp-dart-dap-flutter-hot-reload-on-save t)
;; or
(setq lsp-dart-dap-flutter-hot-restart-on-save t)
```
Thank you!
